### DISCUSSION TITLE
[Question]: Frigate+ labelling and currently unsupported objects

### DISCUSSION THREAD

**Discussion by mrplow** [blakeblackshear/frigate#11409] [Ask A Question] [answered]
### What is your question:

Hi all,

I have a racoon in my neighbourhood that keeps being reported as a dog/cat. Since racoon is not currently a supported label in Frigate+, I've been skipping submitted them.

I do want the notifications for racoons, so I don't want to train the model with false positives when they show up as a dog or cat. But I also don't want to train the model to pretend they're dogs, just in case racoon is added to Frigate+ in the future.

Should I continue to skip submitting them completely?

I'm thinking that I'll just keep the snapshots saved to my computer somewhere so if a racoon label is added later I can easily add them all together.

Thanks for reading and happy detecting!

PS: my very first model works way better than the default Coco. Great work @blakeblackshear

**Comment by NickM-27** [comment, marked as answer]
yes, I also just skip raccoons and birds for the same reason.

**Reply by ehn** [reply]
I sometimes submit birds detected as persons or cats as false positives. Would you say it's better for future base models not to do that?

I have avoided submitting foxes detected as cats as false positives for the reason outlined in this issue.
